src/ansi.rs

```diff
@@ -1574,6 +1577,13 @@ where
             ('E', []) => handler.move_down_and_cr(next_param_or(1) as usize),
             ('F', []) => handler.move_up_and_cr(next_param_or(1) as usize),
             ('G', []) | ('`', []) => handler.goto_col(next_param_or(1) as usize - 1),
+            ('W', [b'?']) => {
+                if next_param_or(0) == 5 {
```

Comment:
```suggestion
            ('W', [b'?']) if next_params_or(0) == 5 => {
```

I think this should work and allow you to avoid explicitly writing the unhandled branch?

Reply:
Done